File: Javascript/1036.js
var input = `10.0 20.1 5.1`;
var lines = input.split(/[\s]/);
let a = lines[0];
let b = lines[1];
let c = lines[2];

if (b * b - 4 * a * c < 0 || 2 * a === 0.0) {
  console.log("Impossivel calcular");
} else {
  console.log(
    "R1 = " + ((-b + Math.sqrt(b * b - 4.0 * a * c)) / (2 * a)).toFixed(5)
  );
  console.log(
    "R2 = " + ((-b - Math.sqrt(b * b - 4.0 * a * c)) / (2 * a)).toFixed(5)
  );
}

File: Python/1036.py
import math
line1 = input().split(" ")
a = float(line1[0])
b = float(line1[1])
c = float(line1[2])

if ((b * b - 4 * a * c) < 0 or 2 * a == 0):
    print("Impossivel calcular")
else:
    print("R1 = %.5f" % ((-b + math.sqrt(b * b - 4 * a * c)) / (2 * a)))
    print("R2 = %.5f" % ((-b - math.sqrt(b * b - 4 * a * c)) / (2 * a)))
